src/components/Player/Pages/Overview/Overview.jsx

```diff
@@ -111,6 +113,9 @@ const Overview = ({
   toggleTurboGames,
   showTurboGames,
   strings,
+  countsData,
+  countsLoading,
+  countsError,
 }) => (
   <OverviewContainer>
     <SummaryContainer
```

Comment:
thinking we should add the rounded border to the summary component as well.  they should look similar.